spec/system/consumer/caching/darkswarm_caching_spec.rb

```diff
@@ -19,6 +19,11 @@
   }
   let(:exchange) { order_cycle.exchanges.outgoing.where(receiver_id: distributor.id).first }
 
+  around do |example|
+    Rails.cache.clear
+    example.run
+  end
```

Comment:
This clears the cache before these examples run but not after. So that specs protect themselves from a stale cache but they don't protect other specs from their own modifications.

These specs use the `caching: true` tag. It may be better to move this code into the spec helper and clear the cache there.